node-runner-cli/setup/DockerSetup.py

```diff
@@ -342,8 +342,6 @@ def confirm_docker_compose_file_changes(docker_config: DockerConfig, autoapprove
     @staticmethod
     def chown_files(docker_config: DockerConfig):
         username = getpass.getuser()
-        run_shell_command(['sudo', 'chown', f'{username}:{username}',
```

Comment:
Wondering if this is still need for those use cases where user already has a key from different machine?  The keyfile may not have right permissions. If we can run this conditionally, it can be based on whether it is existing key or key that is going to be generated later down the line

Reply:
yeah, good point. I'm adding back the command with fail_on_error=False